src/main/java/io/github/thebusybiscuit/slimefun4/implementation/guide/SurvivalSlimefunGuide.java

```diff
@@ -289,7 +290,7 @@ private void displaySlimefunItem(ChestMenu menu, ItemGroup itemGroup, Player p,
             menu.addItem(index, new CustomItemStack(ChestMenuUtils.getNoPermissionItem(), sfitem.getItemName(), message.toArray(new String[0])));
             menu.addMenuClickHandler(index, ChestMenuUtils.getEmptyClickHandler());
         } else if (isSurvivalMode() && research != null && !profile.hasUnlocked(research)) {
-            menu.addItem(index, new CustomItemStack(ChestMenuUtils.getNotResearchedItem(), ChatColor.WHITE + ItemUtils.getItemName(sfitem.getItem()), "&4&l" + Slimefun.getLocalization().getMessage(p, "guide.locked"), "", "&a> Click to unlock", "", "&7Cost: &b" + research.getCost() + " Level(s)"));
+            menu.addItem(index, new CustomItemStack(ChestMenuUtils.getNotResearchedItem(), ChatColor.WHITE + ItemUtils.getItemName(sfitem.getItem()), "&4&l" + Slimefun.getLocalization().getMessage(p, "guide.locked"), "", "&a> " + Slimefun.getLocalization().getMessage(p, "guide.click-unlock"), "", "&7" + Slimefun.getLocalization().getMessage(p, "guide.cost") + " " + research.getCost() + " " + Slimefun.getLocalization().getMessage(p, "guide.level")));
```

Comment:
what if we can use other ways to unlock in the future? i mean this PR: https://github.com/Slimefun/Slimefun4/pull/3942

Reply:
That’s out of scope tho. Creating merge conflicts isn’t the end of the world. And for bigger PRs like that it would be halted by extensive testing anyways

Reply:
Then I guess we don't have to worry about other unlock methods in this pr and just have the other pr handle the changes to that.

Reply:
I'm wondering if the line "Cost: (condition)" should be separated.

Reply:
I would agree with that.

Putting these under a single key is more annoying to untangle later.

Reply:
True. I edited the suggestion. Is this better?

Reply:
not too sure how crowdin works but wouldn't those two empty lines show up as empty strings to translate? wouldnt break anything; just a little confusing